opencog/openpsi/entity-defs.scm

```diff
@@ -0,0 +1,48 @@
+;
+; entity-defs.scm
+;
+; OpenPsi-related entity definitions for use in
+; the interaction dynamics rules.
+;
+;
+; --------------------------------------------------------------
+
+
+; EVENT PREDICATES
+; Event/Stimilus Predicates are assumed to get evaluate to 1 for a single
+; psi-loop step and only for a single psi-loop step whenever a particular
+; instance of the event is detected.
+
+
+;; new face detected
+;(define new-face (Concept "new face"))
+;(define new-face-detected
+	;(Evaluation (stv 0 1)
+	;	(Predicate "psi-event-detected")
+	;	(List
+	;		new-face))
+
+	;(event-detected-eval new-face))
+
+; rework the below with the new Predicate "psi-event-detected"
+;(define speech (Predicate "speech-giving-starts"))
+;(define (speech-giving-starts (psi-create-event-detected-eval speech)))
+
+
+
+; PAU PREDICATES
+; Actually these will be defined somewhere else in the system
```

Comment:
Up until now, there have been no PAU's in opencog, because we routed everything through blender, first.

For hanson robotics, PAU is a synonym for "a motor" -- but perhaps you mean something more general here? some general action to be performed, or some general behavior to carry out?